docs/ansible/roles/debops.phpipam/copyright.rst

```diff
@@ -0,0 +1,4 @@
+Copyright
+=========
+
+.. literalinclude:: ../../../../ansible/roles/debops.phpipam/COPYRIGHT
```

Comment:
This include should be in the `index.rst` file. Check the `debops.netbase` role for a good example.

Reply:
Done